src/components/ReloadInfo.tsx

```diff
@@ -9,6 +9,13 @@ const ReloadInfo: FC = () => {
     localStorage.setItem('showReloadInfo', '0')
   }, [])
 
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setShowReloadInfo(false)
+    }, 10000)
+    return () => clearTimeout(timeoutId)
+  })
```

Comment:
```suggestion
  }, [])
```

The dependencies need to be empty to signalize the component it should do that once.

Reply:
:bulb: changed